stable/kube-slack/values.yaml

```diff
@@ -35,3 +35,7 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+annotations: {}
+  # Specify which annotations must be added to kube-slack pods 
```

Comment:
Not sure why i was mentioned here.
Anyway, seems the is some issue with the values.yaml, which have to be fixed before pr can be merged.

I guess the whitspace after "pods" should be removed...

Reply:
🤦‍♂️